node/service/src/lib.rs

```diff
@@ -816,13 +819,14 @@ where
 	let collator = config.role.is_authority();
 
 	if collator {
-		let env = sc_basic_authorship::ProposerFactory::new(
+		let mut env = sc_basic_authorship::ProposerFactory::new(
 			task_manager.spawn_handle(),
 			client.clone(),
 			transaction_pool.clone(),
 			prometheus_registry.as_ref(),
 			telemetry.as_ref().map(|x| x.handle()),
 		);
+		env.set_soft_deadline(SOFT_DEADLINE_PERCENT);
```

Comment:
Is it the intention to set this to 100%? Wasn't it 50% previously? (This is dev mode...)

Reply:
Oh, it's intentional -- you mentioned it in the description. Does this solve something in particular?

Reply:
Over soft deadline skipping transactions will stop trying to add more transaction to the block. It's equivalent to our previous fix of removing transaction skipping entirely :p